build.sbt

```diff
@@ -585,43 +588,55 @@ lazy val allModules: Seq[ProjectReference] = Seq(
   http,
   kafka,
   `marshallers-jodatime`,
+  `legacy-avro-decimal-compat-model`,
+  `legacy-avro-decimal-compat-protocol`,
+  `legacy-avro-decimal-compat-encoders`,
+  `health-check-unary`,
+  `example-health-client`,
+  `example-health-server-monix`,
+  `example-health-server-fs2`,
   `example-routeguide-protocol`,
   `example-routeguide-common`,
   `example-routeguide-runtime`,
   `example-routeguide-server`,
   `example-routeguide-client`,
+  seed
```

Comment:
`seed` is also an example project, right? Shall we rename it to `example-seed` for consistency?

Reply:
Done in https://github.com/higherkindness/mu-scala/commit/263c3a5fc8426d3ef42772e41a9a9e9c0bb8e5b6